css/catalog-item.css

```diff
@@ -0,0 +1,127 @@
+/* -------------- CATALOG-ITEM SECTION --------------*/
+
+.catalog-item-title {
+    font-size: 44px;
+    line-height: 56px;
+    text-align: center;
+}
+
+.catalog-item-link {
+    font-family: Bakbak One;
```

Comment:
Provide fallback fonts and family types at the end of the font list. 
For example: `font-family: "Roboto", Arial, sans-serif;`